cql3/statements/create_keyspace_statement.cc

```diff
@@ -283,6 +278,13 @@ lw_shared_ptr<data_dictionary::keyspace_metadata> create_keyspace_statement::get
     return _attrs->as_ks_metadata(_name, tm, feat);
 }
 
+::shared_ptr<schema_altering_statement::event_t> create_keyspace_statement::created_event() const {
+    return make_shared<event_t>(
```

Comment:
What did you gain by aliasing the readable "event::schema_change" that we had before, to the generic and mysterious "event_t"?

Reply:
As I noted above, I think in your new implementation there is no longer any need for the new created_event() functions (or the event_t type) - they are only used in one place, and I think that introducing them just makes the code more complicated. They served a purpose when they were a virtual function, and called from the query processor layer, but I don't see what purpose they serve now.

Reply:
I think those changes are making code more readable